MC/CustomGenerators/DPG/Dpmjet_Pileup_Cent.C

```diff
@@ -0,0 +1,53 @@
+#if !defined(__CINT__) || defined(__MAKECINT__)
+//For ROOT6
+R__ADD_INCLUDE_PATH($ALIDPG_ROOT)
+#include "Utils/GetTriggerBCMaskAndMu.C"
+#endif
+
+AliGenerator *GeneratorCustom(){
+
+#if defined(__CINT__)
+  // For ROOT5
+  gROOT->LoadMacro("$ALIDPG_ROOT/Utils/GetTriggerBCMaskAndMu.C++");
+#endif
+
+  Double_t mu=0.001; 
+  TString bcm=GetTriggerBCMaskAndMu(mu);
+  
+  // swap H and L to match requirement of AliGenPileup
+  bcm.ReplaceAll("H","X");
+  bcm.ReplaceAll("L","H");
+  bcm.ReplaceAll("X","L");
```

Comment:
Looks fine, but I would also move these ``...ReplaceAll`` lines in different macros to the GetTriggerBCMaskAndMu

Reply:
Agreed, done